lib/v2/springer/journal.js

```diff
@@ -0,0 +1,72 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+const { CookieJar } = require('tough-cookie');
+const cookieJar = new CookieJar();
+
+module.exports = async (ctx) => {
+    const host = 'https://www.springer.com';
+    const journal = ctx.params.journal;
+    const jrnlUrl = host.concat('/journal/', journal);
```

Comment:
```suggestion
    const jrnlUrl = `${host}/journal/${journal}`;
```
Please use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) or [+](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition) instead of [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#performance)